OWNERS

```diff
@@ -5,5 +5,7 @@ approvers:
   - kaslin
   - mbbroberg
   - parispittman
+  - rajula96reddy 
   - Pensu
   - rajula96reddy
+  - gkarthiks
```

Comment:
A quick thing here. This list is in ascending order by name. Can you please correct the same? Also, if you don't mind can you make a new PR for this. It will be easier to track that way and we can make this PR dedicated for the new feature.

Do check if you are member of `kubernetes-sigs` as it is a per-requisite for the OWNERS file